pyensembl/gtf_parsing.py

```diff
@@ -121,20 +122,30 @@ def _extend_with_attributes(df):
     extra_columns = {}
     column_order = []
     for i, attr_string in enumerate(df.attribute):
+        # TODO: tokenize attribute strings properly
+        # for now, catch mistaken semicolons by replacing "xyz;" with "xyz"
+        # Required to do this since the GTF for Ensembl 78 has
```

Comment:
Yeah, these are semicolons that are incorrectly placed at the end of gene names. Example from the ensembl 78 GTF file:

19	ensembl_havana	CDS	39110053	39110115	.	+	0	gene_id "ENSG00000183760"; gene_version "8"; transcript_id "ENST00000331256"; transcript_version "7"; exon_number "13"; *gene_name "PAPL;"*; gene_source "ensembl_havana"; gene_biotype "protein_coding"; *transcript_name "PAPL;-001"*; transcript_source "ensembl_havana"; transcript_biotype "protein_coding"; tag "CCDS"; ccds_id "CCDS33018"; protein_id "ENSP00000327557"; protein_version "4";